java/repositories.bzl

```diff
@@ -25,472 +25,339 @@
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+load("//toolchains:local_java_repository.bzl", "local_java_repository")
 load("//toolchains:remote_java_repository.bzl", "remote_java_repository")
 
 def java_tools_javac11_repos():
```

Comment:
java_tools_javac11 was an old naming used with v10.x line of java_tools, which had to be in theory released for each javac version (but was only for 11). After that I removed javac11 from the versioning of java_tools.

Please just rename the function from `java_tools_javac11_repos` to `java_tools_repos`.

Thanks for importing v11.x and all the latests JDKs!

Reply:
Done!